README.md

```diff
@@ -1,7 +1,7 @@
 <p align="center"><img src="https://user-images.githubusercontent.com/56656996/99621835-b2d8dc80-29dd-11eb-8183-987e80f8b3a7.png"></p>
 <h1 align="center">DDE Store</h1>
 <h3 align="center">An app store for DDE built with DTK</h3>
-<p align="center"><a href="/translations/README/README.es.md">🇪🇸 Español</a> | <a href="/translations/README/README.de.md">🇩🇪 Deutsch</a> | <a href="/translations/README/README.tr.md">🇹🇷 Türkçe</a> | <a href="/translations/README/README.ar.md">🇸🇦 العربية</a></p>
+<p align="center"><a href="/translations/README/README.es.md">🇪🇸 Español</a> | <a href="/translations/README/README.de.md">🇩🇪 Deutsch</a> | <a href="/translations/README/README.kab.md">Taqbaylit</a> | | <a href="/translations/README/README.tr.md">🇹🇷 Türkçe</a> |<a href="/translations/README/README.ar.md">🇸🇦 العربية</a></p>

```

Comment:
Is there any particular flag emoji that you could be used for the link?

Reply:
@dekzi 
here is the kabyle flag:https://fr.wikipedia.org/wiki/Fichier:Flag-kabyle.svg , but unfortunately there is no emoji with this flag, not yet and I could not create one from the flag.

Reply:
> @dekzi
> here is the kabyle flag:https://fr.wikipedia.org/wiki/Fichier:Flag-kabyle.svg , but unfortunately there is no emoji with this flag, not yet and I could not create one from the flag.

Would the Algerian flag be okay for recognizably? 🇩🇿

Reply:
yes, it will be okay